helpothers/templates/base.html

```diff
@@ -66,6 +67,12 @@
     <script src="{% static "js/jquery.easing.min.js" %}"></script>
     <script src="{% static "js/jquery.fittext.js" %}"></script>
     <script src="{% static "js/wow.min.js" %}"></script>
+    <script src="{% static "js/jquery-ui.min.js" %}"></script>
+    <script src="{% static "js/jspanel/jquery.jspanel.min.js" %}"></script>
```

Comment:
Well those libraries were for implementing popup(div alerts) when a user likes a resource, i wanted it to be like(on the stack overflow page when you like a comment) a div pops up and tells you, something "you just liked this post", that's what i was implementing but i run out of time... guess i'll just remove it.